omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImpl.java

```diff
@@ -144,8 +144,9 @@ public int hashCode() {
 			return result;
 		}
 	}
-	
-	private void initializeResources() {
```

Comment:
Do you mind explaining why you moved this from a private method to the base interface?

Reply:
Oversight. To be fixed in next commit

Reply:
scratch that, it was actually intentional to move it to the base interface because i need to use it in this method as can be seen at https://github.com/mherman22/openmrs-module-webservices.rest/blob/08b817fd98e0c15f81f021d4e856b0c2b598c125/omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerGenerationUtil.java#L408

cc: @dkayiwa

Reply:
Is there any reason why you did not just use `RestService.initialize()`?

Reply:
Fixed